offa.csl

```diff
@@ -0,0 +1,438 @@
+      <term name="reviewed-author" form="verb">rev. of</term>
+      <term name="presented at">talk</term>
+      <term name="accessed">accessed: </term>
+    </terms>
+  </locale>
+  <locale xml:lang="de">
+    <terms>
+      <term name="reviewed-author" form="verb">Rez. zu</term>
+      <term name="presented at">Vortrag</term>
+      <term name="accessed">Zugriff: </term>
+    </terms>
+  </locale>
+  <macro name="short-container-contributors">
+    <choose>
+      <if type="chapter paper-conference" match="any">
+        <group>
+          <choose>
+            <if variable="editor container-title" match="any">
+              <text term="in" text-case="sentence" prefix=". " suffix=": "/>
+              <names variable="editor">
+                <name delimiter="/" delimiter-precedes-et-al="never" delimiter-precedes-last="always" initialize-with="."/>
+                <label form="short" strip-periods="false" prefix=" (" suffix="),"/>
+              </names>
+              <names variable="container-author">
+                <name delimiter="/" delimiter-precedes-et-al="never" delimiter-precedes-last="always" initialize-with="."/>
+              </names>
+              <text macro="collection-title"/>
+            </if>
+          </choose>
+        </group>
+      </if>
+      <else-if type="article article-journal article-magazine article-newspaper" match="any">
+        <text macro="translator"/>
+        <choose>
+          <if match="any" variable="container-title-short">
+            <text variable="container-title-short" prefix=". "/>
+          </if>
+          <else>
+            <text variable="container-title" prefix=". "/>
+          </else>
+        </choose>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="editor">
+    <names variable="editor">
+      <name delimiter="/" delimiter-precedes-et-al="never" delimiter-precedes-last="always" initialize-with=". "/>
+      <label form="short" prefix=" (" suffix=")"/>
+    </names>
+  </macro>
+  <macro name="translator">
+    <choose>
+      <if match="any" variable="translator">
+        <choose>
+          <if match="none" variable="editor">
+            <names variable="translator" prefix=", ">
+              <label form="verb-short" suffix=" "/>
+              <name delimiter="/" delimiter-precedes-et-al="never" delimiter-precedes-last="always" initialize-with="."/>
+            </names>
+          </if>
+        </choose>
+      </if>
+    </choose>
+  </macro>
+  <macro name="contributors">
+    <choose>
+      <if variable="author">
+        <names variable="author">
+          <name delimiter="/" delimiter-precedes-et-al="never" delimiter-precedes-last="always" initialize-with=". "/>
+        </names>
+      </if>
+      <else-if variable="editor">
+        <text macro="editor"/>
+      </else-if>
+    </choose>
+    <choose>
+      <if match="any" variable="title">
+        <choose>
+          <if match="any" variable="editor author">
+            <text value=", "/>
```

Comment:
Couldn't you just put a group with a comma delimiter around contributors and title in the bibliography? Much neater than this